econplayground/templates/base.html

```diff
@@ -68,9 +68,9 @@
 
 {% user_is_instructor request.user as i_am_instructor %}
 <body class="{% block bodyclass %}{% endblock %}" id="{% block bodyid %}{% endblock %}">
-    <div id="skip">
-        <a href="#maincontent">Skip to main content</a>
-    </div>
+    <aside id="skip">
+        <a href="/accounts/login#maincontent">Skip to main content</a>
```

Comment:
Actually - I'm not sure if this was intentional here. This causes the "Skip to main content" button, selectable when you press "Tab" on any page of the site, to always redirect to the login form, even when you're already logged in. I think this should be changed back to the original #maincontent href, unless I'm missing something.